var/spack/repos/builtin/packages/fastmath/package.py

```diff
@@ -0,0 +1,58 @@
+##############################################################################
```

Comment:
Ah, I see. So `fastmath` is just a meta-package that installs other packages. There is a bit of precedence for this. I would take a look at the `xsdk` package that @BarrySmith wrote. It downloads a fake tarball (doesn't matter which) and writes a simple file to the installation directory.

Alternatively, you can ask users to run:
```
$ spack install --only=dependencies fastmath
```
instead so they don't have to worry about the `fastmath` installation.